solarwinds_apm/apm_constants.py

```diff
@@ -40,6 +40,12 @@
     "k8s",
     "uams",
 ]
+INTL_SWO_DEFAULT_RESOURCE_DETECTORS_LAMBDA = [
```

Comment:
Please let me know if more/less should be part of this, like `INTL_SWO_DEFAULT_RESOURCE_DETECTORS`

Reply:
I think the list is good.